.github/workflows/release.yml

```diff
@@ -23,12 +23,38 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+
+      - name: Get dependencies
+        run: make deps
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Test Darwin and Linux
+        run: make coverage
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SYSL_PLANTUML: http://www.plantuml.com/plantuml
+
+      - name: Test Windows
+        run: .\scripts\coverage-windows.bat
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SYSL_PLANTUML: http://www.plantuml.com/plantuml
+
       - name: Validate goreleaser config
         uses: goreleaser/goreleaser-action@master
         with:
           args: check
 
-      - name: Release via goreleaser
+      - name: Login to Docker Hub
```

Comment:
A little bit confused about the docker hub login here. Is it necessary in order for goreleaser to upload the new docker image to docker.io?

Reply:
Yes. goreleaser need this access to push image.